zerver/webhooks/opsgenie/doc.md

```diff
@@ -0,0 +1,16 @@
+{!create-stream.md!}
+
+The integration will automatically use the default stream
+`opsgenie` if no stream is supplied, though you will still
+need to create the stream manually even though it's the
+default.
```

Comment:
Hmm, it is mostly redundant with `{!create-stream.md!}`, except for the info 
> The integration will automatically use the default stream
`opsgenie` if no stream is supplied.

I removed it, but since two other integrations are using the same text snippet, should I make a PR for removing it there as well and add the extra phrase to the macro? (FWIW, the [doc](http://zulip.readthedocs.io/en/latest/integration-docs-guide.html) on this macro is outdated as well, so I can jsut update it at the same time.